src/main/webapp/gameDialogue.js

```diff
@@ -13,8 +13,13 @@
 // limitations under the License.
 const FINAL_STAGE_BUTTON_VALUE = 'COMPLETE PATH';
 const FINAL_STAGE_REDIRECTION = 'CompletedPath.html';
+const MUSIC_ICON= 'musicIcon';
+const UNMUTE_ICON = '<img src="icons/unmute.gif" alt="unmuted icon">';
+const MUTE_ICON = '<img src="icons/mute.png" alt="muted icon">';
 
 var dialogueArray;
+var i;
```

Comment:
What is variable i? It looks unused...

Reply:
it is an iterator for dialogue

Reply:
rename to something more descriptive `dialogueIterator`

Reply:
i was a merging conflict on my par the var i was replaced in the latest commit to master